cmd/cloud_orchestrator/main.go

```diff
@@ -99,9 +99,23 @@ func LoadSecretManager(config *config.Config) secrets.SecretManager {
 
 func LoadOAuth2Config(config *config.Config, sm secrets.SecretManager) *appOAuth2.Helper {
 	var oauth2Helper *appOAuth2.Helper
+	var credential []byte
+	if config.SecretManager.Type == secrets.UnixSMType {
+		secretFilePath := config.SecretManager.UNIX.SecretFilePath
+		var err error
+		credential, err = secrets.GetServiceAccountCredential(secretFilePath)
```

Comment:
If you're referring to the `credential, err :=` part on L105, I declared it outside the if block because the block might not always be executed, but the credential variable could potentially be used on L113.

 Is my understanding correct?

Reply:
Ah, I missed what you said during writing review comments. You're right, never mind.